integration/utils/contLoadGenerator.go

```diff
@@ -462,130 +436,68 @@ func (lg *ContLoadGenerator) sendAddKeyTx(workerID int) {
 }
 
-func (lg *ContLoadGenerator) pushAccountsHappening(workerID int, acc *flowAccount, tx_id string, timeNow time.Time) {
-	lg.log.Trace().Msgf("workerID=%d pushing happening address", workerID)
-	lg.happeningAccounts <- (func() (*flowAccount, string, time.Time) { return acc, tx_id, timeNow })
-}
-
-func (lg *ContLoadGenerator) probeAccountsHappening(workerID int) {
-	channelElements := len(lg.happeningAccounts)
-	lg.log.Trace().Msgf("workerID=%d probing happening address from channel with %d elements", workerID, channelElements)
-	for i := 0; i < channelElements; i++ {
-		acc, tx_id, timeAtSendTx := (<-lg.happeningAccounts)()
-		txIdFile := fmt.Sprintf("/dev/shm/flow-transaction-feedback/%s/%s/%s.txt", tx_id[0:2], tx_id[2:4], tx_id[4:]) // e.g. /dev/shm/flow-transaction-feedback/70/b9/4146a01bc7843dc9547837e19ba0da6899fdd83ae61e2cf2e3fb4555da6a.txt
-		lg.log.Trace().Msgf("workerID=%d probing happening address %x account %d tx_id %s AKA %s", workerID, acc.address.Bytes(), acc.i, tx_id, txIdFile)
-		if _, err := os.Stat(txIdFile); errors.Is(err, fs.ErrNotExist) {
-			// come here if file does not exist; means account is still happening...
-			lg.pushAccountsHappening(workerID, acc, tx_id, timeAtSendTx)
-
-			elapsed := time.Since(timeAtSendTx).Seconds()
-			elapsedSuspicious := 30.0
-			if (elapsed >= elapsedSuspicious) && (elapsed < (1 + elapsedSuspicious)) {
-				// come here to capture transactions which mysteriously have had no feedback after 30 seconds
-				// note: this should only be output once as its age reaches and passes through a certain amount of seconds...
-				lg.log.Warn().Msgf("workerID=%d address %x account %d tx_id=%s has been happening for %f seconds and counting", workerID, acc.address.Bytes(), acc.i, tx_id, elapsed)
-			}
-		} else {
-			// come here if file exists; means account is good to be used again
-			lg.availableAccounts <- acc
-			e := os.Remove(txIdFile)
-			if e != nil {
-				lg.log.Error().Err(err).Msgf("workerID=%d error removing txIdFile=%s", workerID, txIdFile)
-				panic("ERROR: failed to remove txIdFile")
-			}
-		}
-	}
-}
-
 func (lg *ContLoadGenerator) sendTokenTransferTx(workerID int) {
+	log := lg.log.With().Int("workerID", workerID).Logger()
 
-	blockRef, err := lg.blockRef.Get()
-	if err != nil {
-		lg.log.Error().Err(err).Msgf("workerID=%d error getting reference block", workerID)
-		return
-	}
-
-	if 0 == workerID {
-		if lg.feedbackEnabled {
-			// come here if feedbackEnabled so that worker with workerID 0 can probe to see if accounts are no longer happening
-			defer lg.probeAccountsHappening(workerID)
-
-			// come here to have workerID 0 keep track of lowest number of available accounts in channel
-			if len(lg.availableAccounts) < lg.availableAccountsLo {
-				lg.availableAccountsLo = len(lg.availableAccounts)
-				lg.log.Debug().Msgf("workerID=%d discovered new availableAccountsLo=%d of numberOfAccounts=%d", workerID, lg.availableAccountsLo, lg.numberOfAccounts)
```

Comment:
Added `txsInProgress` counter to the log, will also eventually add a corresponding metric to prometheus.

Reply:
After looking at the business logic for the new implementation then I'm worried that the new implementation / counter might be misleading under certain circumstances.

Firstly: The new code replaces the lowest number of available accounts with the highest number of TXs in flight. So the opposite. Which is okay in principle, but I'm not sure what the motivation is switching. With the number of available accounts then this is perhaps easier for the reader / user to process because it should always be > 0. However, with the highest number of TXs in flight then this is perhaps more confusing to the reader / user. Why? Because the reader / user has to do more math to figure out what the ceiling is (TPS * account multiplier) that this stat gets close to, or? And when comparing multiple load tests, it's therefore easier to compare the lowest number of available accounts against zero, than doing the extra math to figure out how close the highest number of TXs in flight get to each ceiling, or?

Secondly: I'm not sure the highest number of TXs in flight is calculated accurately. Why? Consider this example:
* Ceiling is 5,000 TX and TPS is 100.
* Last block received contains 0 TX and shows 4,950 TX in flight at second n+0.
* Second n+1 tries to send 100 TPS but can only send 50; adds 50 to 4,950 = 5,000.
* Next block received contains 200 TX and shows 4,800 TX in flight at second n+2; subtracts 200 from 5,000 = 4,800.

In this example, the loader accounts would be exhausted but the TX in flight count would mislead the reader / user into thinking that the ceiling was never reached... when it in-fact was, or?

The problem is that the sending of TXs and block following are down asynchronously and both in batches -- batches of sending TX, and blocks with batches of TXs finalized -- but the output log is only done after receiving the latest block. Therefore, in the meantime, the atomic counters can get closer to the ceiling than actually reported, or?

My suggestion #1 is to introduce e.g. `txsInProgressHi` which always reports the highest number of TXs in flight so as to avoid the async math reporting issue above.

My suggestion #2 is to switch back to the opposite counter e.g. `availableAccountsLo` so it's easier to glance at this stat and see how close it gets to zero regardless of the TPS or accounts multiplier settings.

My suggestion #3 is to perhaps show both counters, `availableAccounts` (goes up and down using the atomic math and is effectively a snap shot value at the time the next block is finalized) and `availableAccountsLo` (only goes down and always shows the lowest value reached at any point so far)?

Reply:
The main reason for counting inflight transactions instead of the available accounts is because this layer of abstraction does not (and likely should not) know about the total number of accounts.  We should not care about a specific number here since if we the code upstream runs out of accounts to use it screams VERY loudly.

In the future diffs I'll add a counter that in a separate goroutine will emit a gauge of len(accounts) and a separate count on failure to get a new account from the channel.

Reply:
> if we the code upstream runs out of accounts to use it screams VERY loudly

It's true, but even if it never screams and never runs out, then I'd like to know -- preferably by grepping the logs -- how much head room the load test had, i.e. how "healthy" it was running / how close it came to account exhaustion :-)